packages/react-components/react-tags/src/components/TagGroup/useTagGroupContextValues.ts

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import type { TagGroupContextValues, TagGroupState } from './TagGroup.types';
+
+export function useTagGroupContextValues_unstable(state: TagGroupState): TagGroupContextValues {
+  const { size } = state;
+  return React.useMemo(() => ({ tagGroup: { size } }), [size]);
```

Comment:
```suggestion
  return {tagGroup: React.useMemo(() => ({ size }), [size])};
```

nit: no need to context values, just the context value itself